layers/+lang/julia/README.org

```diff
@@ -69,6 +69,9 @@ This layer also includes experimental support for =lsp-mode=. To enable language
 server functionality with this layer, follow [[#lsp][the instructions for installation]],
 and then install this layer with:
 
+Warning: [[https://github.com/JuliaEditorSupport/LanguageServer.jl][ =LanguageServer.jl=]] has not yet been released for Julia 1.0. If using
```

Comment:
You have an extra space here. But I've fixed that for you.

Reply:
Also, I've moved it a little bit down, so it doesn't interfere with enabling instructions.